automata/config/__init__.py

```diff
@@ -48,6 +48,7 @@
 REPOSITORY_NAME = os.getenv("REPOSITORY_NAME", "emrgnt-cmplxty/Automata")
 MAX_WORKERS = int(os.getenv("MAX_WORKERS", 8))
 DATA_ROOT_PATH = os.getenv("DATA_ROOT_PATH", "automata-embedding-data")
+PICKLED_DATA_PATH = os.path.join(DATA_ROOT_PATH, "graphs")
```

Comment:
We could put this in here, but I think we are starting to get a bit too specific w/ what variables we allow in this init file.

Our current approach is to put data paths here 

```python
class EmbeddingDataCategory(PathEnum):
    """
    A class to represent the different categories of configuration options
    Corresponds folders in automata/configs/*
    """

    CODE_EMBEDDING = "code-embedding"
    DOC_EMBEDDING = "doc-embedding-l2"
    INDICES = "indices"
```

We could then introduce a util function like "load_data_path" which can check against these enums and combine with the DATA_ROOT_PATH above.